### PR TITLE
nim: Change to 7z

### PR DESCRIPTION
**Pull request by linsui**
close https://github.com/lukesampson/scoop/issues/3621

Now there is 3 workaround:
- Use develop branch
    `cd $env:SCOOP\apps\scoop\current; git checkout develop; scoop install nim`
- Use pscore
    `scoop install pwsh; pwsh -c 'scoop install nim'`
- Enable UTF8. See https://github.com/ScoopInstaller/Main/pull/391#discussion_r320736623

### CODE REVIEW

File: bucket/nim.json [hunk 1]
```diff
@@ -5,20 +5,22 @@
     "license": "MIT",
     "architecture": {
         "64bit": {
-            "url": "https://nim-lang.org/download/nim-0.20.2_x64.zip",
+            "url": "https://nim-lang.org/download/nim-0.20.2_x64.zip#/dl.7z",
             "hash": "ccd258ae90d2af2eb1042830f1204920e8d44b6947aedb56458110cf9a714207"
         },
         "32bit": {
-            "url": "https://nim-lang.org/download/nim-0.20.2_x32.zip",
+            "url": "https://nim-lang.org/download/nim-0.20.2_x32.zip#/dl.7z",
```
Comment (r15ch13) [inline bucket/nim.json]: Please revert this to `.zip`. This change https://github.com/lukesampson/scoop/pull/3460 will provide this functionality.

Comment (linsui) [reply bucket/nim.json]: Yeah, I know...but how to install nim before https://github.com/lukesampson/scoop/pull/3460 in master?

Comment (r15ch13) [reply bucket/nim.json]: The current manifest works using zip. 😕 

image

Comment (r15ch13) [reply bucket/nim.json]: Stop! I'm using the develop branch. :grin:

Comment (r15ch13) [reply bucket/nim.json]: Still works with master branch:
image

Comment (linsui) [reply bucket/nim.json]: Strange... It does not work for me. I got the same error as https://github.com/lukesampson/scoop/issues/3621 .

2019-09-02 20_35_13-Windows PowerShell

Comment (r15ch13) [reply bucket/nim.json]: What is the output of `$PSVersionTable`?

Comment (linsui) [reply bucket/nim.json]: I get the same error and an empty directory. 

```
D:\..\cache ❯❯❯ $psversiontable

Name                           Value
----                           -----
PSVersion                      5.1.18362.145
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.18362.145
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

Comment (akkuman) [reply bucket/nim.json]: @r15ch13 When I run `Microsoft.PowerShell.Archive\Expand-Archive -Path ".\nim#0.20.2#https_nim-lang.org_download_nim-0.20.2_x64.zip" -DestinationPath nim -Force` from the scoops cache directory, I got the same error as https://github.com/lukesampson/scoop/issues/3621

Comment (akkuman) [reply bucket/nim.json]: @linsui What terminal do you use? It looks cool. Haha

Comment (linsui) [reply bucket/nim.json]: https://github.com/akkuman It's powershell with colortool and oh-my-posh :joy: .
@r15ch13 It works well in pscore 6.2.2 .

Comment (r15ch13) [reply bucket/nim.json]: hm, can't reproduce it in any way.

image

Comment (Rvn0xsy) [reply bucket/nim.json]: Maybe Chinese windows 10

Comment (linsui) [reply bucket/nim.json]: @Rvn0xsy Maybe not, but related. I enable UTF8 and it works well now.

**Pull request closed without merging**